drivers/mulesoft/booking_api.cr

```diff
@@ -14,7 +14,7 @@ class MuleSoft::BookingsAPI < PlaceOS::Driver
     time_zone:          "Australia/Sydney",
     ssl_key:            "private key",
     ssl_cert:           "certificate",
-    ssl_auth_enabled:   true,
+    ssl_auth_enabled:   false,
```

Comment:
What's the reason to disable SSL by default?

Reply:
`on_load` runs with default settings (before settings are set) causing the driver to crash due to invalid ssl_key/ssl_cert